noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -613,64 +616,44 @@ contract Token {
         finalization_payload.emit();
     }
 
-    /// We need to use different randomness for the user and for the fee payer notes because if the randomness values
```

Comment:
Have we got this text retained somewhere else? It feels like a useful learning that could get lost.

Reply:
We don't but you have a point.

@critesjosh would you know of a place in the docs where this could fit? With this PR it no longer applies to this code so I nuked it (because we now work only with 1 partial note instead of 2).

Reply:
@jzaki is working on fee docs. I'll sync with him about it